### DISCUSSION TITLE
Hotkey: Profiles & connections

### DISCUSSION THREAD

**Discussion by m33rc4t** [Eugeny/tabby#5984] [Q&A] [answered]
Hello

I would like to bind a Key to the button "Profiles & connections". So that I do not have to press with the mouse the button. Is this hotkey bind missing or can I not find it?
Thanks

![2022-03-22_08-13_Tabby_PBFTmgaBas](https://user-images.githubusercontent.com/14199266/159427775-dac1f513-053e-4be7-9022-9ea40f64c8dd.png)

**Comment by Eugeny** [comment, marked as answer]
That hotkey is called `profile-selector`.

**Reply by m33rc4t** [reply]
Thank you, that was the hotkey I was looking for!

**Reply by m33rc4t** [reply]
As a supplement. There are by default three hotkeys (new-tab, reopen-tab, profile-selector) which have the same key assignment. This leads to a conflict.

**Comment by lawweiliang** [comment]
@n0NamETDH  
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)

**Reply by m33rc4t** [reply]
Sorry, I forgot about that 😅
